File: SerialPortDemo/trunk/CS/Serial/DataReceivedEventArgs.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Serial
{
    public class DataReceivedEventArgs : EventArgs
    {

        public DataReceivedEventArgs(string data)
        {
            _data = data;
        }

        private string _data;
        public string Data
        {
            get
            {
                return _data;
            }
            set
            {
                _data = value;
            }
        }
    }
}

File: SerialPortDemo/trunk/VB/Serial/DataReceivedEventArgs.vb
Imports System.Collections.Generic
Imports System

Public Class DataReceivedEventArgs
    Inherits EventArgs

    Public Sub New(ByVal data As String)
        _data = data
    End Sub

    Private _data As String
    Public Property Data() As String
        Get
            Return _data
        End Get
        Set(ByVal Value As String)
            _data = Value
        End Set
    End Property

End Class
